src/shared/lib/validateParameters.ts

```diff
@@ -0,0 +1,25 @@
+import { validate, ParameterValidationError } from 'parameter-validator';
+import * as _ from 'lodash';
+
+export interface URLValidationResult {
+    isValid:Boolean;
+    message:string | null
+}
+
+export default function(params: { [k:string]: string }, rules:any): URLValidationResult {
```

Comment:
what is the purpose of `rules`? I don't see a reference to `rules` in the function body.